### DISCUSSION TITLE
Key bindings for virtual gears controller (bluetooth or wired)

### DISCUSSION THREAD

**Discussion by rvrooyen** [cagnulein/qdomyos-zwift#375] [Ideas]
Hi. Was wondering how the key bindings will work for changing the setting for virtual gears. If I used a wired controller that can be programmed to specify a specific key binding, could QZ use that to increase / decrease the virtual gears setting?

**Comment by cagnulein** [comment]
hi! i was thinking about bluetooth remote, like a Playstation controller or the cheap bleutooth remote for the car. In that way, with the up/down volume button we can change the gears tile!
